src/examples/3_arm_and_gripper.ipynb

```diff
@@ -198,8 +199,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "reachy.r_arm.goto_joints(elbow_at_90_deg)\n",
-    "reachy.l_arm.goto_joints(elbow_at_90_deg)"
+    "reachy.r_arm.goto_joints(r_elbow_at_90_deg)\n",
+    "reachy.l_arm.goto_joints(l_elbow_at_90_deg)"
```

Comment:
The example below inverse kinematics does not work

I had to change for something like
```
import numpy as np
target = np.identity(4)
target[0][3] = 0.4
target[1][3] = 0.1
target[2][3] = -0.3
target
```

The elbow is set for top grasping. Maybe we can add an example about that also? What do you think?